build-hpx-kokkos.sh

```diff
@@ -20,7 +20,8 @@ if [[ ! -d ${DIR_SRC} ]]; then
 	cd ..
 	git clone https://github.com/STEllAR-GROUP/hpx-kokkos.git hpx-kokkos
 	cd hpx-kokkos
-	git checkout ce63b0630c739a4df899749ed67baa867862f3fc
+	#git checkout ce63b0630c739a4df899749ed67baa867862f3fc
+	git checkout master
```

Comment:
We should add some new variable in config.sh and provide the kokkos-hpx version from there to keep it consistent

https://github.com/STEllAR-GROUP/OctoTigerBuildChain/blob/main/config.sh#L44